gi_notifier/runner.py

```diff
@@ -0,0 +1,32 @@
+from asyncio import Event
+
+from apscheduler.schedulers.asyncio import AsyncIOScheduler
+from pytz import timezone
+from telegram.ext import ApplicationBuilder, ChatMemberHandler, CommandHandler
+
+from . import config
+from .handler.chat_handler import handle_bot_added
+from .handler.command_handler import handel_resinplan, handle_start
+from .scheduler import scheduled_task
+
+
+async def runner():
+    application = ApplicationBuilder().token(config.BOTTOKEN).build()
+    application.add_handler(CommandHandler("start", handle_start))
```

Comment:
Can you ensure the oneshot nature of this command please? There is no point in retriggering the `/start` command again or letting folks do it for that matter once the initial setup is through.